CHANGELOG.md

```diff
@@ -18,7 +27,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Update README and repository description.
 - Clean up CHANGELOG.
-
-[Unreleased]: https://github.com/rokwire/safer-illinois-app/compare/v2.3.5+205...HEAD
```

Comment:
These will be good to have to see all the changes (code, commits, etc.) between two adjacent releases.